source/whats-new.txt

```diff
@@ -38,6 +38,11 @@ What's New in 1.21
 The {+library-short+} v1.21 release includes the following features,
 improvements, and fixes:
 
+- Introduces the Aggregation Builder, an API to build aggregation
+  pipelines in a more type-safe and PHP-native way. To learn more and
```

Comment:
```suggestion
  pipelines in a more type-safe way. To learn more and
```

"PHP-native" reads like filler, as I don't think it elicits a common meaning among readers. If anything, the original syntax where stages are expressed using PHP arrays could be considered the most "native".